src/scenes/Projects/Files/ProjectFilesList.tsx

```diff
@@ -114,16 +115,29 @@ const ProjectFilesListWrapped: FC = () => {
   const navigate = useNavigate();
   const { projectId } = useParams();
   const formatDate = useDateTimeFormatter();
+  const { enqueueSnackbar } = useSnackbar();
 
   const { openFilePreview } = useFileActions();
 
   const {
     loading: directoryLoading,
+    canRead: canReadRootDirectory,
     project,
     directoryId,
     rootDirectoryId,
   } = useProjectCurrentDirectory();
 
+  if (canReadRootDirectory === false) {
+    enqueueSnackbar(
+      `You don't have permission to view files for this project`,
```

Comment:
Something permanent would probably be better. The card could be replaced with typography. Breadcrumbs would still be shown to allow easy back.

Another similar case: https://github.com/SeedCompany/cord-field/blob/e9310f0eb169b9ca9b03029a109afeed06370c27/src/scenes/Languages/Detail/LanguageDetail.tsx#L223-L226